.github/triage_replies.md

```diff
@@ -91,3 +93,9 @@ The AWX team.
 
 ## Try latest version
 - Hello, this issue pertains to an older version of AWX. Try upgrading to the latest version and let us know if that resolves your issue.
+
+
+## Other
+
+### Denying requests to add modules to awx-ee
+- Hello. While we understand how this could be useful, we feel that this could end up with a fairly bloated Execution Environment. The awx-ee contains the collections and dependencies needed for supported AWX features to function. Anything beyond that will require you to build your own EE. For information on how to do that, see https://ansible-builder.readthedocs.io/en/stable/
```

Comment:
Do we want to do two of these? One for PRs/Issues and one for a triage reply? i.e. "The reason the job is reporting the module as missing is because the default awx-EE contains only the collection and dependencies needed to ...."